.github/DOCUMENTATION_TEMPLATE.md

```diff
@@ -0,0 +1,65 @@
+# How to conttribute to MFTF docs
+
+MFTF documentation is kept within the /docs/ folder in this repository.
+We welcome contributions to the documentation.
+This page describes the process for submitting documentation and serves as a template for a properly written content.
+
+The contribution workflow for the Magento functional testing framework (MFTF) documentation is the same as submitting code.
+
+1. Make a branch from the `develop` branch in the [MFTF repo][].
+1. Make edits/additions/deletions as needed.
+1. Submit your PR to the `develop` branch.
+
+Once submitted, a member of the documentation team will review and merge it.
+We will inform you if it needs any additional processing.
+
+Any changes to the Table of Contents will need to be made in the regular [devdocs repo][].
+
+## H2 Heading - blank line before and after, capitalize first word only
```

Comment:
The section is empty.
`H2` is obvious for a writer but can be confusing for a contributing developer.
Please consider applying my previous comments.

Reply:
Ok.